tests/test_cirq.py

```diff
@@ -166,7 +158,8 @@ def test_one_qubit_gates(backend, gate_name, nqubits, ndevices):
     targets = random_active_qubits(nqubits, nactive=1)
     qibo_gate = getattr(gates, gate_name)(*targets)
     cirq_gate = [(getattr(cirq, gate_name), targets)]
-    assert_gates_equivalent(backend, qibo_gate, cirq_gate, nqubits, ndevices)
+    if backend.__class__.__name__ != "CupyBackend":
```

Comment:
What is the problem with the `CupyBackend` here? I believe these tests used to pass with it.

Reply:
There are some `Cirq` fucntions that don't work with `cupy.ndarray` inputs due to the way they're implemented. Same thing for `EagerTensor`s. I even proposed a change to `Cirq` [here](https://github.com/quantumlib/Cirq/pull/6076) to fix some of these issues.